### ISSUE TITLE
philisophical foundation - broken link

### ISSUE THREAD

**Issue by shishini** [JordanMartinez/purescript-jordans-reference#536]
```
Section:
  Preface
    Intended Audience

Issue Type:
  Bug
```

## Body

Broken link: https://jordanmartinez.github.io/purescript-jordans-reference-site/content/01-FP-Philosophical-Foundations

**Comment by JordanMartinez** [comment]
This is not a bug. There's isn't a Readme file for that folder: https://github.com/JordanMartinez/purescript-jordans-reference/tree/latestRelease/01-FP-Philosophical-Foundations.

**Comment by JordanMartinez** [comment]
Ah, now I see the problem. The link itself should point to the first file rather than the folder itself. When reading through the repo via GitHub, one can see the folder. But when doing it via the mdbook version, it'll just show a 404.

**Comment by JordanMartinez** [comment]
Closed in 93dc97790052c40d3226df666907e2b860c88040.
